consensus/istanbul/core/prepare_test.go

```diff
@@ -81,3 +84,38 @@ func TestCore_sendPrepare(t *testing.T) {
 		mockCtrl.Finish()
 	}
 }
+
+func BenchmarkMsgCmp(b *testing.B) {
+	getEmptySubject := func() istanbul.Subject {
+		return istanbul.Subject{
+			View: &istanbul.View{
+				Round:    big.NewInt(0),
+				Sequence: big.NewInt(0),
+			},
+			Digest:   common.HexToHash("1"),
+			PrevHash: common.HexToHash("2"),
+		}
+	}
+	s1, s2 := getEmptySubject(), getEmptySubject()
+
+	// Worst
+	b.Run("reflect.DeepEqual", func(b *testing.B) {
+		for i := 0; i < b.N; i++ {
+			reflect.DeepEqual(s1, s2)
+		}
+	})
+
+	// Better
+	b.Run("cmp", func(b *testing.B) {
```

Comment:
I have no idea where "gotest.tools/assert/cmp" is from. is the import related to this improvement?

Reply:
Yes, but I forgot to remove its comparison since it cannot be used in production env.